### PR TITLE
feat: add persisting auth

### PR DESCRIPTION
**Pull request by anik31**

### CODE REVIEW

File: src/features/auth/authSlice.js [hunk 1]
```diff
@@ -18,6 +19,16 @@ export const loginUser = createAsyncThunk("auth/login", async(credentials, { rej
   }
 });
 
+export const verifyUser = createAsyncThunk("auth/verify", async(encodedToken, { rejectWithValue })=>{
+  try{
+    const {data} = await axios.post("/api/auth/verify", {encodedToken});
+    console.log(data);
```
Comment (deekshithmd) [inline src/features/auth/authSlice.js]: you can remove...this console if everything working fine

File: src/features/auth/authSlice.js [hunk 2]
```diff
@@ -89,6 +119,7 @@ const authSlice = createSlice({
         })
       },
       [signupUser.rejected]: (state, action) => {
+        state.isAuthLoading = false;
         state.status = "error";
         state.error = action.payload;
         toast({
```
Comment (deekshithmd) [inline src/features/auth/authSlice.js:124]: Code looks clean and understandable...you can proceed to merge

**Review by deekshithmd — APPROVED** [review]

File: src/features/auth/authSlice.js (hunk 1 above)
Comment (anik31) [reply src/features/auth/authSlice.js]: Thanks for pointing it out, will remove it.

**Review by sainath7878 — APPROVED** [review]
You can go ahead and merge the PR. Nothing to suggest.

**Pull request merged**